src/components/pages/DetailRecordPage/DetailRecordPage.js

```diff
@@ -0,0 +1,159 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import Axios from '../../../service/axios';
+import { useParams } from 'react-router-dom';
+
+import { DetailRecordPageStyled } from './DetailRecordPage.styled'
+import HighlightTable from '../../elements/HighlightTable/HighlightTable'
+import AttorneyInput from '../GenerationPage/GenerationInput/AttorneyInput';
+import PetitionerInput from '../GenerationPage/GenerationInput/PetitionerInput';
+import InputSection from '../../elements/Input/InputSection';
+import { Button } from '../../elements/Button';
+import PetitionList from '../GenerationPage/PetitionList';
+
+const DEFAULT_STATE_LABEL = { label: 'NC', value: 'NC' };
+
+const REQUIRED_FIELDS = [
+    'name',
+    'address1',
+    'city',
+    'state',
+    'zipCode'
+  ];
+
+const ButtonStyled = styled(Button)`
+  font-size: 1.5rem;
+  margin: 1rem;
+`;
+
+function PageButton({ label, onClick, disabled }) {
+    return (
+      <ButtonStyled onClick={onClick} disabled={disabled}>
+        {label}
+      </ButtonStyled>
+    );
+  }
+
+
+function DetailRecordPage() {
```

Comment:
Just want to make sure. You removed the route to this page, right? I think we'll eventually use this separate page, but that should be in a subsequent update

Reply:
Yes, I removed the route.